data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "entity_type": "work",
+  "uniform_name": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
+  "description": "A commentary on the six days of Creation. The first of its kind, this style of commentary, called Hexaemeron, flourished in late antiquity, and later commentaries drew on Basil's as a model.",
+  "authority_files": [{
+      "id": "https://haf.vhmml.org/work/320756617003",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron"
+    },
+    {
+      "id": "https://viaf.org/viaf/181934772",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. | Hexaemeron"
+    }
+  ],
+  "associated_dates": [{
+    "type": "creation",
+    "date_human_readable": "mid-4th c. CE",
+    "date_normalized": "0325/0375"
+  }],
+  "language": "Greek",
+  "genre": ["Theological Works", "Commentary, Biblical", "Commentary, Old Testament", "Commentary, Genesis"],
+  "associated_entities": [{
+      "type": "person",
+      "relation": "author",
+      "id": "https://sinaimanuscripts.library.ucla.edu/person/55",
+      "notes": []
+    }
```

Comment:
off-hand list of options we could have for person relations: author, editor (redactor, translator, etc.), character, commemorated, contra, subject of, etc.; could have places: creation, subject of, set in (but these are getting way out of scope for SDP…). This could also be where we might have literary tradition style relations between ancient versions (translations, recensions, etc.) -- work-to-work relationships.